paasta_tools/cli/cmds/security_check.py

```diff
@@ -45,14 +46,22 @@ def add_subparser(subparsers):
 
 
 def perform_security_check(args):
+    """It runs a few security-checks including:
+    - shellshock: is the version of bash running inside the container vulnerable to shellshock?
+    - docker container: is the container running as root?
+    - packages: does the container has the latest packages installed? If not, the output will contain the name of the
+    package that needs to be updated and to which version.
```

Comment:
Having said that I guess you did this so that it appears nicely in readthedocs? Maybe it would be okay to include some sort of link to our internal wiki from the alert rather than readthedocs...